internal/ordering_test.go

```diff
@@ -509,8 +509,8 @@ policies:
   - path: {{printf "%v/%v" .Dir "configmap.yaml"}}
   - path: {{printf "%v/%v" .Dir "configmap.yaml"}}
 `,
-			wantFile: "",
-			wantErr:  "policyDefaults may not specify both consolidateManifests and orderManifests",
+			wantFile: "testdata/ordering/three-ordered-manifests.yaml",
+			wantErr:  "",
 		},
 		"orderManifests in policy and consolidateManifests is false": {
```

Comment:
If we can update this description as well, something like
```suggestion
		"orderManifests is true in policy and consolidateManifests is unset": {
```